### PR TITLE
Don't render the overlay if drawer is not open

### LINKED ISSUE #15: touch input not recognized (react-native v0.15.x)
Since I updated to react-native version v0.15.0 from v0.13.2, I've had the problem that only the swipe gesture (to open the menu) is recognized but NOT the touch input on the child views of the drawer layout. As I explored the code I saw that the overlay is always on top of the child views and consuming all my touch inputs.

Now I wonder if that's a problem in my code, some side effect maybe OR if there were some changes with react native causing this issue. 

```
  <View style={{flex: 1, backgroundColor: 'transparent'}} {...this._panResponder.panHandlers}>

    <Animated.View style={styles.main}>
      {this.props.children}
    </Animated.View>

    <Animated.View style={[styles.overlay, animatedOverlayStyles]} />

    <Animated.View style={[styles.drawer, dynamicDrawerStyles, animatedDrawerStyles]}>
      {this.props.renderNavigationView()}
    </Animated.View>
  </View>
```

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Hi there,

I fixed #15 by simply not rendering the overlay as long as the drawer is not shown at all. Afterwards not beeing able to click is the intended behaviour.

### CODE REVIEW

**Comment by DanielMSchmidt** [comment]
@brentvatne @mfmendiola If one of you has some spare time to review this PR I would be so happy. Otherwise, I will just merge it in about 10 hours, I tested it quite a lot on my local setup.

File: src/DrawerLayout.ios.js [hunk 1]
```diff
@@ -89,6 +89,7 @@ export default class DrawerLayout extends React.Component {
       extrapolate: 'clamp',
     });
     const animatedDrawerStyles = { transform: [{ translateX: drawerTranslateX }] };
+    const drawerShown = (openValue._value > 0);
```
Comment (mfmendiola) [inline src/DrawerLayout.ios.js]: Hm, why are you using `openValue._value` here as opposed to just `openValue`?

**Comment by mfmendiola** [comment]
One nit; otherwise, lgtm!

File: src/DrawerLayout.ios.js (hunk 1 above)
Comment (DanielMSchmidt) [inline src/DrawerLayout.ios.js]: Okay, I didn't thought I might do this, but it works :+1: First time with the animated API for me :rocket:

**Pull request merged**